docs/runbooks/VirtApiRESTErrorsBurst.md

```diff
@@ -2,8 +2,8 @@
 
 ## Meaning
 
-For the last 10 minutes or longer, over 80% of the REST calls made to `virt-api`
-pods have failed.
+More than 80% of REST calls have failed in the `virt-api` pods in the last
+5 minutes.
```

Comment:
In a PR by @sradco (https://github.com/kubevirt/monitoring/pull/229/files), this was recently updated to the "For the last 10 minutes or longer..." text, replacing the older "More than 80%..." phrasing. Does sradco's change need to be made downstream (in the openshift/runbooks runbooks and the downstream docs)?

Reply:
Yes. The text I used is the correct one and should be updated in the d/s runbooks

Reply:
make sense, reverted all